internal/rps/message.go

```diff
@@ -70,7 +70,7 @@ func (p Payload) createPayload(dnsSuffix string, hostname string, amtTimeout tim
 	var err error
 	wired, _ := p.AMT.GetLANInterfaceSettings(false)
 	if wired.LinkStatus != "up" {
-		log.Warn("link status is down, unable to active AMT in Admin Control Mode (ACM)")
+		log.Warn("link status is down, unable to activate AMT in Admin Control Mode (ACM)")
```

Comment:
This message will be used for all the rpc commands including maintenance (which is not 'activate') so if I try to sync the clock of update the DNS, it will still say 'unable to activate'.
main -> rps.prepareInitialMessage -> payload.CreateMessageRequest -> p.createPayload -> p.AMT.GetLANInterfaceSettings

Reply:
perfect thing to address in a separate PR